### PR TITLE
daemon: add "isWindows" const

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
hm; think I missed a commit in this branch; let me look it up

**Review by AkihiroSuda — APPROVED** [review]

File: daemon/daemon.go [hunk 1]
```diff
@@ -74,7 +74,10 @@ import (
 )
 
 // ContainersNamespace is the name of the namespace used for users containers
-const ContainersNamespace = "moby"
+const (
+	isWindows           = runtime.GOOS == "windows"
```
Comment (justincormack) [inline daemon/daemon.go]: you could just set this in daemon_windows.go and daemon_unix.go as constants and not import `runtime` at all, its kind of even more constanty.

Comment (thaJeztah) [reply daemon/daemon.go]: ah, good point we have separate files for both; yes, I'll update

Comment (thaJeztah) [reply daemon/daemon.go]: updated 👍

**Review by justincormack — APPROVED** [review]

**Pull request merged**